plugins/woocommerce-admin/client/marketplace/stylesheets/_variables.scss

```
@@ -31,4 +31,6 @@ $gutenberg-gray-700: $gray-700;
 $gutenberg-gray-900: $gray-900;
 $mauve-light-12: $gray-900;
 $woo-purple-50: #7f54b3;
+$wp-gray-0: #F6F7F7;
```

For this, you can either set it to reference `$gray-0` which is from the _colors.native.scss file imported at the top or directly use that `$gray-0` value in your styles.

```suggestion
$wp-gray-0: $gray-0;
```

I had this same issue initially, which is why the variables file has variables referencing other variables. The problem comes from the colour names used in Figma – they are prefixed with `wp-` but actually just refer to the base styles.

Hint: if the colour is `gutenberg-` that is a colour from [_colors.scss](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_colors.scss). If the colour is `wp-` then it is from [_colors.native.scss](https://github.com/WordPress/gutenberg/blob/trunk/packages/base-styles/_colors.native.scss) instead.